graphstorm-processing/graphstorm_processing/data_transformations/dist_transformations/dist_noop_transformation.py

```diff
@@ -95,15 +122,17 @@ def str_list_to_float_vec(string_list: Optional[List[str]]) -> Optional[List[flo
                     for column in self.cols
                 ]
             )
-            return input_df
         else:
-            return input_df.select(
+            input_df = input_df.select(
```

Comment:
Any reason we remove the return statement here?

Reply:
The logic is changed below, either side of the if/else statement now just a assigns a value to input_df, which is the conditionally passed through the truncate function before returning the DF.